pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md

```diff
@@ -261,6 +293,22 @@ This error happens when not parsed messages and actions were found from the vend
 ## Description
 This error happens when the bot has an invalid action body and Lex is unable to parse it. It can be a wrong parameter, a value, or a missing parameter. Even though it is showing as an error in the Third Party Bots, the bot gently ignores unknown actions and the conversation will continue.
 
+## Transfer Action Error
```

Comment:
Please follow the format I provided above errors.

Reply:
done